docs/analytics/jenkins.md

```diff
@@ -210,14 +212,14 @@ Also, ensure these repositories are cloned and readable by the jenkins user:
 ## jenkins_env
 
 Running pipeline jobs requires some environment variables to be set - these are listed in
-[jenkins_env](resources/jenkins_env).  Variables include the [S3 buckets created above](#s3-buckets) and others:
+[jenkins_env](resources/jenkins_env).  Variables include the [S3 buckets created above](#pipeline-s3-buckets) and others:
```

Comment:
Please change the [referenced section](https://github.com/open-craft/openedx-deployment/pull/7/files#diff-7f455a44f79c304247b8df1cdf8dc667R171)'s title to `Pipeline S3 buckets`.